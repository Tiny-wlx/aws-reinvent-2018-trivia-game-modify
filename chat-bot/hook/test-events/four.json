{
  "messageVersion": "1.0",
  "invocationSource": "DialogCodeHook",
  "userId": "ozw40htm8dvj0n1lk9ugt6gja961h4dl",
  "sessionAttributes": {
    "started": "true",
    "currentQuestion": "4",
    "currentScore": "100",
    "currentSlot": "four"
  },
  "requestAttributes": null,
  "bot": {
    "name": "TriviaGame",
    "alias": "$LATEST",
    "version": "$LATEST"
  },
  "outputDialogMode": "Text",
  "currentIntent": {
    "name": "LetsPlay",
    "slots": {
      "fifteen": null,
      "nine": null,
      "six": null,
      "one": 2012,
      "seven": null,
      "three": "blah",
      "two": "blah",
      "thirteen": null,
      "eight": null,
      "fourteen": null,
      "four": "blah",
      "sixteen": null,
      "twelve": null,
      "eleven": null,
      "ten": null,
      "five": null
    },
    "confirmationStatus": "None"
  },
  "inputTranscript": "Let's play"
}
