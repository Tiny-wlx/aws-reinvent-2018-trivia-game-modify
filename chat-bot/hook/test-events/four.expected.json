{
  "sessionAttributes": {
    "started": "true",
    "currentQuestion": 5,
    "currentScore": 100,
    "currentSlot": "five"
  },
  "dialogAction": {
    "type": "ElicitSlot",
    "intentName": "LetsPlay",
    "slots": {
      "fifteen": null,
      "nine": null,
      "six": null,
      "one": 2012,
      "seven": null,
      "three": "blah",
      "two": "blah",
      "thirteen": null,
      "eight": null,
      "fourteen": null,
      "four": "blah",
      "sixteen": null,
      "twelve": null,
      "eleven": null,
      "ten": null,
      "five": null
    },
    "slotToElicit": "five",
    "message": {
      "contentType": "PlainText",
      "content": "Incorrect! The correct answer is \"2014\". Moving on the \"The Main Event\" category. For 100 points: What webinar helps people learn about re:Invent leading up to the big event?"
    }
  }
}
